aiohttp/client.py

```diff
@@ -544,6 +545,9 @@ async def _request(
                     if auth is None:
                         auth = auth_from_url
 
+                    if auth is None:
+                        auth = get_auth_from_netrc(url)
```

Comment:
If I'm not mistaken, this is doing blocking I/O and it's happening on every request which could a serious impact on performance.

I think if we want this, then the netrc object would need to be loaded on client instantiation, rather than on every request.

Reply:
Good point, fixed in [`b612685` (#9133)](https://github.com/aio-libs/aiohttp/pull/9133/commits/b6126859fdc2dfd47fb7f52024254ede9469395a) through reading the netrc file once at client initialisation.